book.py

```diff
@@ -1,12 +1,12 @@
 class Book:
     def __init__(self, title, author, pages):
-        self.title = title
+        self.title = titl
         self.author = author
         self.pages = pages
 
```

Comment:
This new method is helpful for retrieving formatted task details. Consider adding error handling for invalid task IDs.